rasa/shared/importers/importer.py

```diff
@@ -329,7 +329,7 @@ async def get_domain(self) -> Domain:
         )
 
         existing_domain = existing_domain.merge(domain_with_retrieval_intents)
-        existing_domain.check_missing_responses()
```

Comment:
This was a bug not noticed before, API v3 used in v2 🤔 Wonder when it would be run into.